cmd/unalias.rb

```diff
@@ -13,7 +13,7 @@ def unalias_args
 
         Remove aliases.
       EOS
-      min_named 1
+      named_args(min: 1)
```

Comment:
```suggestion
      named_args :alias, min: 1
```

And change the `usage_banner` method to be :

```ruby
       description <<~EOS
         Remove aliases.
       EOS
```

This allows the usage string to be automatically generated

Reply:
The `:alias` parameter (will be assigned to `type`) seems to be optional if any of `number:` `max:` `min:` is specified?

Reply:
Adding a type (`:alias`) allows the usage banner to be generated appropriately and automatically

Reply:
Well I see. It is assigned to `@named_args_type`. My fault. 🤦